### PR TITLE
feat: add missing attachements on SensitivityArea APIs

### PR DESCRIPTION
**Pull request by lpofredc**
PR to fix missing attachments in SensitiveAreas APIs.

Tests updated to check if attachments are in API responses.

### CODE REVIEW

**Review by Chatewgne — APPROVED** [review]

**Review by Chatewgne — CHANGES_REQUESTED** [review]
Please add Prefetch in SensitiveAreaViewSet queryset (here : https://github.com/GeotrekCE/Geotrek-admin/blob/master/geotrek/api/v2/views/sensitivity.py#L35) for performances as follow

```
        queryset = sensitivity_models.SensitiveArea.objects.existing() \
            .filter(published=True) \
            .select_related('species', 'structure') \
            .prefetch_related('species__practices',
                              Prefetch('attachments',
                                       queryset=Attachment.objects.select_related('license', 'filetype', 'filetype__structure'))) \
            .alias(geom_type=GeometryType(F('geom')))
```

**Comment by Chatewgne** [comment]
Also please add an entry to describe this PR in latest changelog https://github.com/GeotrekCE/Geotrek-admin/blob/master/docs/changelog.rst

File: geotrek/sensitivity/views.py [hunk 1]
```diff
@@ -116,75 +139,133 @@ class SensitiveAreaAPIViewSet(APIViewSet):
...
             if not trek.is_public():
                 raise Http404
             qs = trek.published_sensitive_areas
-            qs = qs.prefetch_related('species')
-            qs = qs.annotate(geom_type=GeometryType(F('geom')))
-            qs = qs.annotate(geom2d_transformed=Case(
-                When(geom_type='POINT', then=Transform(Buffer(F('geom'), F('species__radius'), 4), settings.API_SRID)),
-                When(geom_type='POLYGON', then=Transform(F('geom'), settings.API_SRID))
-            ))
+            qs = qs.prefetch_related("species")
```
Comment (Chatewgne) [inline geotrek/sensitivity/views.py:198]: Please avoid reformatting this whole file

**Review by Chatewgne — CHANGES_REQUESTED** [review]

File: geotrek/sensitivity/views.py (hunk 1 above)
Comment (lpofredc) [reply geotrek/sensitivity/views.py:198]: Which formatters (python and jinja/html/js) to use and where can i find formatter config in this project ? These changes occured by using autopep8.

**Comment by lpofredc** [comment]
Changes done. 

There will probably be conflicts to manage with other PR I purposed due to simultaneous changes on serializers and viewsets.

**Review by camillemonchicourt — APPROVED** [review]

**Comment by submarcos** [comment]
Hello @lpofredc, on a effectué quelques modifs et créé un début de guide contribution. Peux-tu rebaser et repush ton code pour lancer les tests ? On va essayer de faire en sorte que tout se lance bien dans la CI pour les contributions externes

Merci

File: geotrek/sensitivity/mixins.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+from django.conf import settings
+from django.contrib.gis.db.models.functions import Transform
+from django.db.models import F, Case, When, Prefetch
+from geotrek.common.models import Attachment
+from geotrek.common.functions import GeometryType, Buffer, Area
+
+logger = logging.getLogger(__name__)
+
+class SensitiveAreaQueryset:
```
Comment (submarcos) [inline geotrek/sensitivity/mixins.py:11]:
```suggestion

class SensitiveAreaQueryset:
```

File: geotrek/sensitivity/mixins.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    """Mixin used for to properly querying SensitiveAreas"""
+
+    def get_queryset(self, *args, **kwargs):
+        qs = super(SensitiveAreaQueryset, self).get_queryset()
```
Comment (submarcos) [inline geotrek/sensitivity/mixins.py:15]:
```suggestion
        qs = super().get_queryset()
```

File: geotrek/sensitivity/mixins.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                    ),
+                    When(
+                        geom_type__in=("POLYGON", "MULTIPOLYGON"),
+                        then=Transform(F("geom"), settings.API_SRID),
+                    ),
+                )
+            )
+            .annotate(area=Area("geom2d_transformed"))
+            .order_by("-area")
+            )
```
Comment (submarcos) [inline geotrek/sensitivity/mixins.py:49]:
```suggestion
        )
```

**Pull request merged**